src/uniprotkb/components/protein-data-views/__tests__/__snapshots__/DiseaseInvolvementView.spec.tsx.snap

```diff
@@ -141,19 +141,7 @@ exports[`DiseaseInvolvement should render DiseaseInvolvement 1`] = `
               </a>
             </td>
             <td>
-              in 
-              <a
-                class="external-link"
-                href="https://www.ncbi.nlm.nih.gov/snp/rs13447459"
-                rel="noopener"
-                target="_blank"
-              >
-                dbSNP:rs13447459
-                <test-file-stub
-                  data-testid="external-link-icon"
-                  width="12.5"
-                />
-              </a>
+              in AD; supposed to be rendered with the disease
```

Comment:
Just double checking if the link is supposed to be removed here?

Reply:
Re-added example data in the mock that triggers rendering a link